engine/class_modules/sc_death_knight.cpp

```diff
@@ -6846,20 +6866,32 @@ struct vampiric_blood_buff_t : public buff_t
 
 struct vampiric_blood_t : public death_knight_spell_t
 {
+  int gorefiends_domination_energize_amount;
   vampiric_blood_t( death_knight_t* p, const std::string& options_str ) :
     death_knight_spell_t( "vampiric_blood", p, p -> spec.vampiric_blood )
   {
```

Comment:
```suggestion
  int gorefiends_domination_energize_amount;
  vampiric_blood_t( death_knight_t* p, const std::string& options_str ) :
    death_knight_spell_t( "vampiric_blood", p, p -> spec.vampiric_blood ),
    gorefiends_domination_energize_amount( 0 )
  {
```

Reply:
I know that this isn't strictly necessary since all usages are inside ptr checks, but in general we try to always initialize variables, at least in class modules.